test/e2e/api_test.go

```diff
@@ -47,6 +47,16 @@ func TestRegisterEvents(t *testing.T) {
 		User:      user,
 	})
 	assert.NoError(t, err)
+	iecMetricsEvt, err := json.Marshal(&api.InternalErrorCountMetricsEvent{
```

Comment:
Please use a non-error case of metric events. Reporting errors to test the API could be misunderstood as real errors.